features/bevragen/persoon/verblijfplaats/protocollering-vraag.feature

```diff
@@ -0,0 +1,67 @@
+#language: nl
+
+Functionaliteit: protocollering van de vraag voor verblijfplaats
+
+  Rule: Met fields gevraagde velden worden geprotocolleerd als de elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën.
+    Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden opgenomen.
+    # het is nog niet duidelijk in hoeverre de volgorde van rubrieken een bepaalde volgorde of sortering aan moet houden (voor testbaarheid is dat wel nodig)
+
+    Abstract Scenario: Met fields vragen om <fields> wordt vastgelegd als zoekrubrieken <zoek rubrieken>
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam            | waarde     |
+      | anummer (01.10) | 8468619643 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                               | waarde   |
+      | gemeente van inschrijving (09.10)  | 0518     |
+      | functie adres (10.10)              | W        |
+      | datum aanvang adreshouding (10.30) | 20190714 |
+      | datum ingang geldigheid (85.10)    | 20190714 |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                                       | waarde           |
+      | gemeente_code                              | 0518             |
+      | straatnaam (11.10)                         | Borgesiusstraat  |
+      | naam openbare ruimte (11.15)               | Borgesiusstraat  |
+      | huisnummer (11.20)                         | 103              |
+      | huisletter (11.30)                         | b                |
+      | huisnummertoevoeging (11.40)               | 2                |
+      | postcode (11.60)                           | 2497BV           |
+      | woonplaats (11.70)                         | Scheveningen     |
+      | identificatiecode verblijfplaats (11.80)   | 0518010000747448 |
+      | identificatiecode nummeraanduiding (11.90) | 0518200000747446 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | <fields>                        |
+      Dan is protocollering opgeslagen met de volgende gegevens
+      | anummer    | request_zoek_rubrieken |
+      | 8468619643 | <zoek rubrieken>       |
+
+      Voorbeelden:
```

Comment:
Op de meeste plaatsen in deze tabel moet de code 081110 gewijzigd worden in 081115. Voor het bepalen van het type verblijfPlaats moet immers niet 'straatnaam (11.10)' worden gebruikt maar 'naam openbare ruimte (11.15)'.